anchore_engine/configuration/localconfig.py

```diff
@@ -168,43 +181,48 @@ def load_defaults(configdir=None):
     return localconfig
 
 
-def load_policy_bundle_paths(src_dir=None):
+def load_policy_bundle_paths(src_dirs=None):
     global localconfig
 
     default_bundle_name = "anchore_default_bundle.json"
 
     # Get the dir containing policy bundles to put in the config
     policy_bundles_dir = localconfig["policy_bundles_dir"]
 
+    # This value will typically == None, outside of automated tests
+    if src_dirs == None:
+        src_dirs = policy_bundle_source_dirs()
+
     try:
-        if policy_bundles_dir is not None:
+        if (
+            src_dirs is not None
+            and len(src_dirs) > 0
+            and policy_bundles_dir is not None
+        ):
             policy_bundles_dir_full_path = os.path.join(
                 localconfig["service_dir"], policy_bundles_dir
             )
             if not os.path.exists(policy_bundles_dir_full_path):
                 os.mkdir(policy_bundles_dir_full_path)
 
-            if src_dir == None:
-                src_dir = os.path.join(
-                    resource_filename("anchore_engine", "conf/bundles/")
-                )
             policy_bundles = []
-            for file_name in os.listdir(src_dir):
-                try:
-                    file = os.path.join(policy_bundles_dir_full_path, file_name)
-                    policy_bundles.append(
-                        {
-                            "active": file_name == default_bundle_name,
-                            "bundle_path": file,
-                        }
-                    )
-                    copy_config_file(file, file_name, src_dir)
-                except Exception as e:
-                    logger.warn(
-                        "Policy bundle {} not found, unable to load. Exception: {}".format(
-                            file_name, e
+            for src_dir in src_dirs:
+                for file_name in os.listdir(src_dir):
+                    try:
+                        file = os.path.join(policy_bundles_dir_full_path, file_name)
+                        policy_bundles.append(
+                            {
+                                "active": file_name == default_bundle_name,
+                                "bundle_path": file,
+                            }
+                        )
+                        copy_config_file(file, file_name, src_dir)
+                    except Exception as e:
+                        logger.warn(
```

Comment:
using `logger.warn` eats the traceback. Are we trying to fully obscure the traceback?